test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/PackIntegrationTest.cs

```diff
@@ -51,7 +78,7 @@ public class PackIntegrationTest : MSBuildIntegrationTestBase
             Assert.NuspecContains(
                 result,
                 Path.Combine("obj", Configuration, "ClassLibrary.1.0.0.nuspec"),
-                $"<file src=\"{Path.Combine("bin", Configuration, "netcoreapp2.0", "ClassLibrary.Views.dll")}\" " +
+                $"<file src=\"{Path.Combine(Project.DirectoryPath, "bin", Configuration, "netcoreapp2.0", "ClassLibrary.Views.dll")}\" " +
```

Comment:
Is this right for a .nuspec? Is it **really** supposed to have the absolute path? are the other paths absolute paths?

Reply:
```xml
> cat .\obj\Debug\ClassLibrary.1.0.0.nuspec
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
  <metadata>
    <id>ClassLibrary</id>
    <version>1.0.0</version>
    <authors>ClassLibrary</authors>
    <owners>ClassLibrary</owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <description>Package Description</description>
    <dependencies>
      <group targetFramework=".NETCoreApp2.0">
        <dependency id="Microsoft.AspNetCore.Razor.Runtime" version="2.1.0-preview2-t000" exclude="Build,Analyzers" />
        <dependency id="Microsoft.AspNetCore.Razor.Test.MvcShim" version="2.1.0-preview2-t000" exclude="Build,Analyzers" />
      </group>
    </dependencies>
  </metadata>
  <files>
    <file src="D:\work\Razor\test\testapps\ClassLibrary\bin\Debug\netcoreapp2.0\ClassLibrary.Views.dll" target="lib\netcoreapp2.0\ClassLibrary.Views.dll" />
    <file src="D:\work\Razor\test\testapps\ClassLibrary\bin\Debug\netcoreapp2.0\ClassLibrary.dll" target="lib\netcoreapp2.0\ClassLibrary.dll" />
  </files>
</package>
```

Reply:
Bump?